wunderground_upload.yaml

```diff
@@ -190,23 +204,23 @@ variables:
     {{ sensor_humidity if sensor_humidity == '' else (states(sensor_humidity) | float(default=0)) }}
   windspeedmph: >
     {% set unit_of_measurement = state_attr(sensor_windspeed, 'unit_of_measurement') %}
-    {{ sensor_windspeed if sensor_windspeed == '' or unit_of_measurement != 'km/h'
+    {{ (states(sensor_windspeed) | float(default=0)) if sensor_windspeed == '' or unit_of_measurement != 'km/h'
```

Comment:
Why is this change needed? The current code works, just like other sensors (for example humidity)

Reply:
Sensors that aren't native km/h (i.e. mph) won't get cast to a float otherwise. That makes it impossible to graph

Reply:
I see the problem. Only this results in a value of 0 if the sensor is not provided, which you don't want to upload. This is fixed in commit 19f366d